public/app/plugins/datasource/grafana-azure-monitor-datasource/query_ctrl.test.ts

```diff
@@ -224,11 +224,15 @@ describe('AzureMonitorQueryCtrl', () => {
         };
       });
 
-      it.skip('should set the options and default selected value for the Aggregations dropdown', () => {
+      it('should set the options and default selected value for the Aggregations dropdown', () => {
```

Comment:
WDYT about using `async` `await` instead @sunker?
```typescript
      it('should set the options and default selected value for the Aggregations dropdown', async () => {
        await queryCtrl.onMetricNameChange();
        expect(queryCtrl.target.azureMonitor.aggregation).toBe('Average');
        expect(queryCtrl.target.azureMonitor.aggOptions).toEqual(['Average', 'Total']);
        expect(queryCtrl.target.azureMonitor.timeGrains).toEqual([
          { text: 'auto', value: 'auto' },
          { text: 'PT1M', value: 'PT1M' },
          { text: 'P1D', value: 'P1D' },
        ]);
      });
```

Reply:
I like that, but in that case I think we should replace all callbacks in the whole file, and preferably in all tests for this datasource. Will have to do that at some other point.

Reply:
Okidoki, I'll leave that for future improvements.